Source/WebCore/layout/formattingContexts/inline/InlineContentConstrainer.h

```diff
@@ -43,10 +43,16 @@ class InlineContentConstrainer {
 
 private:
     friend struct SlidingWidth;
+    enum class PreviousLineEndsWithLineBreak {
+        None,
```

Comment:
What does 'none' mean in this context?  No previous line? (in which case, I think it should be optional<>)